<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NameWizardPage</class>
 <widget class="QWizardPage" name="NameWizardPage">
  <attribute name="title">
    <string>Name of the loan</string>
  </attribute>
  <layout class="QVBoxLayout">
    <item>
      <widget class="QLabel" name="TextLabel4_2">
        <property name="text">
          <string>
How do you want to call this loan? Examples for names are 'car loan', 'school loan', 'home owner loan'.</string>
        </property>
        <property name="alignment">
          <set>Qt::AlignTop</set>
        </property>
        <property name="wordWrap">
          <bool>true</bool>
        </property>
      </widget>
    </item>
    <item>
      <spacer name="Spacer15_2">
        <property name="sizeHint">
          <size>
            <width>16</width>
            <height>56</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <property name="margin">
          <number>0</number>
        </property>
        <property name="spacing">
          <number>6</number>
        </property>
        <item>
          <widget class="QLabel" name="TextLabel5">
            <property name="text">
              <string comment="@label loan name">Name</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="KMyMoneyLineEdit" name="m_nameEdit"/>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer19">
        <property name="sizeHint">
          <size>
            <width>16</width>
            <height>56</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
    <item>
      <widget class="QLabel" name="m_generalReceiverText">
        <property name="text">
          <string notr="true">TextLabel6</string>
        </property>
        <property name="wordWrap">
          <bool>false</bool>
        </property>
      </widget>
    </item>
    <item>
      <layout class="QHBoxLayout">
        <item>
          <widget class="QLabel" name="m_receiverLabel">
            <property name="text">
              <string notr="true">TextLabel7</string>
            </property>
            <property name="wordWrap">
              <bool>false</bool>
            </property>
          </widget>
        </item>
        <item>
          <widget class="KMyMoneyPayeeCombo" name="m_payeeEdit"/>
        </item>
      </layout>
    </item>
    <item>
      <spacer name="Spacer21">
        <property name="sizeHint">
          <size>
            <width>16</width>
            <height>56</height>
          </size>
        </property>
        <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="orientation">
          <enum>Qt::Vertical</enum>
        </property>
      </spacer>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <customwidgets>
  <customwidget>
   <class>KMyMoneyPayeeCombo</class>
   <extends>QWidget</extends>
   <header>kmymoneypayeecombo.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
 </tabstops>
</ui>
